clients/drcachesim/tests/allasm_repstr.asm

```diff
@@ -53,6 +53,13 @@ _start:
         cld
         rep      movsb
 
+        // Test page-spanning accesses.
+        lea      rcx, page_str
+        // Somehow the assembler is adding 4 to the displacements here
```

Comment:
That's interesting. So rcx and rcx-4 will read the same address?

Reply:
No, they would read addresses 4 bytes apart, but +4 from what was written.  It is very strange.  It adds the load size to the displacement.  Happens with GNU as 2.38.  Added the version to the comment.  It must be some kind of bug, maybe with the Intel syntax flag; may try to see if it's known offline.